### PR TITLE
make console.log messages optional

### PR DESCRIPTION
**Pull request by actuallymentor**
I like being in total control over my console when implementing a 3rd party module like this one.

This PR checks if `options.verbose` is set to false, if nothing is set the evaluations result in `true` meaning the default behaviours do not change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,8 @@ Then put this in your code:
         optional, if is shown before Table of content, such like copyright pages. default: false;
     - `filename`:
         optional, specify filename for each chapter, default: undefined;
+- `verbose`:
```
Comment (pedrosanta) [inline README.md]: Hi @actuallymentor, needs 4 spaces indentation here.

File: README.md [hunk 2]
```diff
@@ -85,6 +85,8 @@ Then put this in your code:
...
+    specify whether or not to console.log progress messages, default: true.
```
Comment (pedrosanta) [inline README.md]: Perhaps the default should be `false`, no? Wdyt @cyrilis?

File: lib/index.js [hunk 3]
```diff
@@ -187,17 +187,17 @@
     render() {
       var self;
       self = this;
-      console.log("Generating Template Files.....");
+      if( !self.options.verbose ) console.log("Generating Template Files.....");
```
Comment (pedrosanta) [inline lib/index.js]: Shouldn't this (and the ones below) be `if(self.options.verbose) console.log("Generating Template Files.....");` instead?

**Comment by actuallymentor** [comment]
Hello @pedrosanta!

I think I fixed your above comments, let me know if everything is in order now.

**Comment by pedrosanta** [comment]
Look good @actuallymentor, thanks! (I have a couple comments on another PR of you, I'll do them asap and publish a new version after we merge them in.)

**Comment by actuallymentor** [comment]
That's good to hear.

Looking forward to your comments on the other PR!

**Pull request merged**